<launch>

  <arg name="planning_plugin" value="stomp_moveit_interface/StompPlanner" />

  <arg name="planning_adapters" value="default_planner_request_adapters/AddTimeParameterization default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints" />

  <param name="planning_plugin" value="$(arg planning_plugin)" />
  <param name="request_adapters" value="$(arg planning_adapters)" />

  <param name="start_state_max_bounds_error" value="0.1" />

  <rosparam command="load" file="$(find pr2_moveit_config)/config/kinematics.yaml"/>
  <rosparam command="load" file="$(find stomp_moveit_interface)/config/pr2/stomp_planning.yaml"/>

</launch>
